hazelcast/src/test/java/com/hazelcast/genericrecord/AbstractGenericRecordTest.java

```diff
@@ -314,4 +268,108 @@ public void testInconsistentClassDefinition() {
         map.put(2, inConsistentNamedRecord);
     }
 
+    @NotNull
+    private MainPortable createMainPortable() {
+        NamedPortable[] nn = new NamedPortable[2];
+        nn[0] = new NamedPortable("name", 123);
+        nn[1] = new NamedPortable("name", 123);
+        InnerPortable inner = new InnerPortable(new byte[]{0, 1, 2}, new char[]{'c', 'h', 'a', 'r'},
+                new short[]{3, 4, 5}, new int[]{9, 8, 7, 6}, new long[]{0, 1, 5, 7, 9, 11},
+                new float[]{0.6543f, -3.56f, 45.67f}, new double[]{456.456, 789.789, 321.321}, nn,
+                new BigDecimal[]{new BigDecimal("12345"), new BigDecimal("123456")},
```

Comment:
nit: it might be a good idea to use a `BigDecimal` with non-zero scale as one of test values.